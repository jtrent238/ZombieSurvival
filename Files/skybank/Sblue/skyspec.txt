;Sblue 
;SUNLIGHT DIRECTION VECTOR
SunDirection	= 100,100,100
;SUNLIGHT STRENGH MULTIPLIER (DEFAULT 100)
SunStrength     = 100
;SUN COLOR (RGB)
SunColor        = 200,221,211
;SUN ROTATION
SunRotation	= 130
;SKYSCROLL SPEED
SkyScrollX      = 10
SkyScrollZ      = 10