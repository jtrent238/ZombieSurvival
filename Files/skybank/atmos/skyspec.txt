;atmos
;SUNLIGHT DIRECTION VECTOR
SunDirection	= 100,100,100
;SUNLIGHT STRENGH MULTIPLIER (DEFAULT 100)
SunStrength     = 100
;SUN COLOR (RGB)
SunColor        = 126,174,208
;SUN ROTATION
SunRotation	= 335
