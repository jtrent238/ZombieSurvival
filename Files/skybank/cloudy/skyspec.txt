;Blue Cloudy Sky 
;SUNLIGHT DIRECTION VECTOR
SunDirection	= 100,100,100
;SUNLIGHT STRENGH MULTIPLIER (DEFAULT 100)
SunStrength     = 100
;SUN COLOR (RGB)
SunColor        = 151,206,240
;SUN ROTATION
SunRotation	= 130