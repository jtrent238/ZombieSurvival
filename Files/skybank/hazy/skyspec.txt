;Hazy Sky 
;SUNLIGHT DIRECTION VECTOR
SunDirection	= 100,100,100
;SUNLIGHT STRENGH MULTIPLIER (DEFAULT 100)
SunStrength     = 100
;SUN COLOR (RGB)
SunColor        = 203,214,211
;SUN ROTATION
SunRotation	= 130