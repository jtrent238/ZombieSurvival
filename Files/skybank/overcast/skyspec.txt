;Overcast Sky 
;SUNLIGHT DIRECTION VECTOR
SunDirection	= 100,100,100
;SUNLIGHT STRENGH MULTIPLIER (DEFAULT 100)
SunStrength     = 100
;SUN COLOR (RGB)
SunColor        = 194,206,213
;SUN ROTATION
SunRotation	= 130