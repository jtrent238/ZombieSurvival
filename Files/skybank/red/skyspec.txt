;Clear Sky 
;SUNLIGHT DIRECTION VECTOR
SunDirection	= 100,50,100
;SUNLIGHT STRENGH MULTIPLIER (DEFAULT 100)
SunStrength     = 100
;SUN COLOR (RGB)
SunColor        = 180,86,60
;SUN ROTATION
SunRotation	= 130