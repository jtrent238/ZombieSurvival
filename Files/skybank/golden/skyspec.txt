;golden
;SUNLIGHT DIRECTION VECTOR
SunDirection	= 100,100,100
;SUNLIGHT STRENGH MULTIPLIER (DEFAULT 100)
SunStrength     = 100
;SUN COLOR (RGB)
SunColor        = 255,214,133
;SUN ROTATION
SunRotation	= 312
